### ISSUE TITLE
Adaptive learning rate for linear regression

### ISSUE THREAD

**Issue by jacowp357** [online-ml/river#1722]
Is there a way to adapt the learning rate of the linear regression model over time?

Like a exponential decay of sorts?

**Comment by jacowp357** [comment]
There seems to be a "scheduler" and "InverseScaling" function, but I'm really struggling to get this to work. See my pipeline below:

```
regressor_model = compose.Pipeline(
    ('features', compose.TransformerUnion(
            ('ordinal_date', compose.FuncTransformer(get_ordinal_date)),
            ('15min', compose.FuncTransformer(get_15min_interval_distances)),
            ('hour', compose.FuncTransformer(get_hour_distances)),
            ('dayOfWeek', compose.FuncTransformer(get_day_of_week_distances)),
            ('dayOfMonth', compose.FuncTransformer(get_day_of_month_distances)),
            ('holiday', compose.FuncTransformer(get_holiday)), 
    )),
    ('scale', preprocessing.StandardScaler()),
    ('lin_reg', linear_model.LinearRegression(
        optimizer=optim.SGD(lr=optim.schedulers.InverseScaling(learning_rate=0.0005)),
        intercept_lr=0,
    ))
)
```

Or is there a way to manually set the learning rate during each step?
